modules/navier_stokes/src/actions/NSFVAction.C

```diff
@@ -273,7 +278,7 @@ NSFVAction::validParams()
       std::vector<MooseFunctorName>(),
       "The ambient temperature for each block in 'ambient_convection_blocks'.");
 
-  params.addParam<CoupledName>(
```

Comment:
can you delete this typedef of CoupledName then

Reply:
Hmm there are still a decent number of places in (P)INSFV and consequently in the action where we use coupled variables

Reply:
Changing that could be more follow on work